Posit Infrastructure/How to Use the R Studio Profiling Tool.md

```diff
@@ -0,0 +1,63 @@
+# How to Use the R Studio Profiling Tool
```

Comment:
Should this be Posit?
Also should the name be changed a little to something like "How to use the Posit memory and CPU profiling tool". Just to many of PHS profiling is about profiling the data rather then the system usage.

Reply:
Nope, RStudio (technically the name has no spaces) is correct - Posit the company makes Posit Workbench, which 'hosts' RStudio the IDE, which has the profiling tool 😂
```suggestion
# How to Use the RStudio Profiling Tool
```